src/main/java/org/apache/sling/jcr/repoinit/impl/RepositoryInitializerFactory.java

```diff
@@ -177,5 +192,10 @@ protected void applyOperations(Session session, List<Operation> ops, String logM
         }
     }
 
+    
+    protected boolean continueOnError() {
```

Comment:
nitpick, now that the property is named "fail on error" I'd use the same terminology here and call the method failOnError